src/components/Tooltip/__tests__/tooltip.interactions.ts

```diff
@@ -0,0 +1,36 @@
+import { expect } from "@storybook/jest";
+import { Screen } from "@testing-library/react";
+import { userEvent, waitFor } from "@storybook/testing-library";
+import { ComponentDefaultTestId, getTestId } from "../../../tests/test-ids-utils";
+import { ONE_LINE_ELLIPSIS_TEST_ID } from "../../Text/__tests__/text-tests-constants";
+
+export async function isTestTooltipShowOnHover(
```

Comment:
This function is doing a multiple stuff
1. trigger show
2. assertion
3. trigger hide

so I think it's better to call it ("triggerTooltipAndAssert")

Reply:
Assert is not clear enough... I can change to testHoverTooltipTrigger?